src/js/common.ml

```diff
@@ -5,6 +5,11 @@ open Mo_config
 module Js = Js_of_ocaml.Js
 module Sys_js = Js_of_ocaml.Sys_js
 
+(** Extra flags from the [moc] CLI *)
+let moc_args = Mo_args.inclusion_args
+  @ Mo_args.warning_args
+  @ Mo_args.error_args
```

Comment:
Any other flags we should include here?

Reply:
The current set of flags looks good for now.